style.css

```diff
@@ -1,2 +1,199 @@
-body { font-family: "Montserrat", serif; }
+body { 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6044074)